webapp/src/containers/Rhs/Rhs.container.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useCallback} from 'react';
+
+import {Spinner} from '@brightscout/mattermost-ui-library';
+
+import {pluginApiServiceConfigs} from 'constants/apiService.constant';
+
+import usePluginApi from 'hooks/usePluginApi';
+
+import {getConnectedState, getIsRhsLoading, getSnackbarState} from 'selectors';
+
+import {Snackbar} from 'components';
+
+import {ConnectAccount} from './views/ConnectAccount';
+import {LinkedChannels} from './views/LinkedChannels';
+import {ConnectedAccount} from './views/ConnectedAccount';
+
+export const Rhs = () => {
+    const {state, getApiState} = usePluginApi();
+    const {connected} = getConnectedState(state);
+    const {isRhsLoading} = getIsRhsLoading(state);
+
+    const {isOpen} = getSnackbarState(state);
+
+    const {data: linkedChannels} = getApiState(pluginApiServiceConfigs.getLinkedChannels.apiServiceName);
+
+    // NOTE: Commented out on purpose.This is part of Phase-II
+    // const isAnyChannelLinked = useMemo(() => Boolean((linkedChannels as ChannelLinkData[])?.length), [linkedChannels]);
+    const isAnyChannelLinked = false;
+
+    const getRhsView = useCallback(() => {
```

Comment:
I don't think the `useCallback` is actually needed here since we're calling the function immediately, and the work done by the function is simple. Tracking if the dependencies of this have changed is likely just as much work for the browser as it would to write this without the hook